quilt/test/build.yml

```diff
@@ -1,7 +1,11 @@
 ---
 contents:
   dataframes:
-    csv: [csv, data/10KRows13Cols.csv]
-    tsv: [tsv, data/10KRows13Cols.tsv]
-    xls: [xlsx, data/10KRows13Cols.xlsx]
-  README: [raw, data/README.md]
+    csv: 
+      file: data/10KRows13Cols.csv
```

Comment:
OK. I see from the code below that the transform can be inferred from the file extension. This convention might be confusing to users wanting to include raw files. As I understand it, no transform specified means "choose the default", but to explicitly choose to include a file as is, they must specify "transform: raw."